src/SDKs/CustomProviders/Management.CustomProviders/generate.ps1

```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "customproviders/resource-manager" -AutoRestVersion "latest"
```

Comment:
Was the code generated using this file? If yes, it should have created a .txt file, please commit this file.